djstripe/models.py

```diff
@@ -320,6 +328,9 @@ class Customer(StripeObject):
     date_purged = models.DateTimeField(null=True, editable=False)
 
     objects = CustomerManager()
+    
+    allow_multiple_subscriptions = ALLOW_MULTIPLE_SUBSCRIPTIONS
+    retain_canceled_subscriptions = RETAIN_CANCELED_SUBSCRIPTIONS
```

Comment:
Ah I got you. As long as the new behavior doesn't introduce any bugs, I'm fine with it. Depending on what @pydanny says, it might be best to always retain_canceled_subscriptions